### PR TITLE
css-brochure-form | Samuel Cho

### PR DESCRIPTION
**Pull request by SamuelCho1**
Used HTML and CSS to build a custom form.
Screen Shot 2021-01-20 at 7 44 30 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Sam! Great work so far, please see my notes. Also, you might want to try removing duplication from the CSS by using grouped selectors where applicable, such as with font family.

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
+body > div > div > form > div {
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+div.content {
+  margin-left: 25%;
```
[the remark this answers is not in the document]
Comment (codingcodymiller) [reply css-brochure-form/styles.css:7]: This has still not been addressed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Sam! You missed a comment from my review, please take a look 👀

**Comment by SamuelCho1** [comment]
Screen Shot 2021-01-23 at 11 08 17 AM
Screen Shot 2021-01-23 at 11 08 52 AM

**Review by codingcodymiller — APPROVED** [review]
Nice work!!! 🎉

**Pull request merged**